code-workflow-logic/about-python-functions.md

```diff
@@ -40,8 +40,8 @@ For example:
 
 ```python editable=true slideshow={"slide_type": ""}
 x = 5
-# The print statement is a function that provides output.
-print(x)
+# abs is a function that provides output.
```

Comment:
I felt it was a little dangerous to conflate console output with function output. IMO notebooks already conflate this a lot, but the text displaying is not the return of the print function (that is just `None`).